zulip/zulip
# Add option to display some custom fields on small profile

Some organisations would like to have some custom profile fields displayed in
the small profile, instead of being shown only in the full profile, which is one
extra click away. This commit offers a way of choosing fields to be displayed in
the small profile. #10135 

**Testing Plan:** 
1. Add a custom field and mark it to be displayed on small profile
2. Set a value for it
3. Verify that the value appears on the small profile

**GIFs or Screenshots:** 

![image](https://user-images.githubusercontent.com/315678/59192911-95290600-8ba1-11e9-9d59-8dc2c9c8b30b.png)
